setup.py

```diff
@@ -54,6 +58,7 @@
         'pyopenssl',
         'ndg-httpsclient',
         'pyasn1',
+        'future',
```

Comment:
It's already in there, but it is pinned to a different version. I'm not sure that really matters, but I can bump it if you want.